packages/trader/src/App/Constants/platform-config.js

```diff
@@ -0,0 +1,28 @@
+import { localize } from 'App/i18n';
+import { routes }   from 'Constants/index';
+
+const is_bot = /^\/bot/.test(location.pathname);
```

Comment:
can we export `is_bot` later we will have `is_mt5` and `is_trader` so its accessible everywhere?